### DISCUSSION TITLE
How to get terminal as I can do with Docker?

### DISCUSSION THREAD

**Discussion by goodidea-kp** [cri-o/cri-o#5851] [Q&A] [answered]
`crictl exec -ti <CONTAINER_ID> \bin\bash` is not giving me a terminal

How do get this convenient tool for debugging work in crio?
Please provide more examples hot to map on 'old' Docker style commands.

**Comment by goodidea-kp** [comment, marked as answer]
`sudo crictl exec 36a58b561fba0 sh -c 'cd /var/log/nginx; cat host.access.log'`

**Reply by haircommander** [reply]
if /bin/bash exists in the container, then it sounds like you just needed to add sudo. That command up there works too
